pages/common/nms.md

```diff
@@ -0,0 +1,20 @@
+# nms
+
+> A cool (piped) text decryption effect.
+> More information: <https://github.com/bartobri/no-more-secrets>.
+
+- Decrypt "Hello, World!" after a keystroke:
+
+`echo "Hello, World! | nms"`
+
+- Decrypt the output of `ls -la` automatically:
```

Comment:
This sort of 'automatic' behavior, while maybe not default to this command, is generally expected from most commands, so 'automatically' is more likely to confuse the user.
```suggestion
- Decrypt the output of `ls -la`:
```

Reply:
I agree with you that people usually expect this behavior by default, and for instance in the next examples I think you're right about ignoring "automatically" in their description. However, if in the first example "nms" is used, and in this one "nms -a" is used, I would say people will be more confused if not explained what's the difference.

Reply:
Ah, good point.  In that case, what do you think showing the 'automatic' behavior first and the delayed behavior later?

Reply:
Honestly, I think we should leave it as is. It is a problem of the command itself, as you say the default behavior should be automatically taking place, with an option to make it non automatic, however this is reversed, leading to confusion.

IMO there's no simple non-confusing way to fix it here, if we do as you say (first showcasing the automatic behavior, then not) it will a bit awkward. Still, if you insist I have no problem changing it.

It would be nice to hear other opinions as well.

Reply:
>It would be nice to hear other opinions as well.

I agree.  Applying the other suggestions would make this one much more noticeable ;)